features/taxation/managing_tax_rates/tax_rate_validation.feature

```diff
@@ -81,3 +81,12 @@ Feature: Tax rate validation
         When I remove its zone
         And I try to save my changes
         Then I should be notified that zone has to be selected
+
+    @ui
+    Scenario: Trying to add a new tax rate with negative amount
+        Given I want to create a new tax rate
+        When I name it "Food and Beverage Tax Rates"
```

Comment:
```suggestion
        When I want to create a new tax rate
        And I name it "Food and Beverage Tax Rates"
```

Reply:
It is consistent with other scenarios like [this one](https://github.com/Sylius/Sylius/blob/dc727c51ea51de39c7312769436f51610e12eb98/features/taxation/managing_tax_rates/tax_rate_unique_code_validation.feature#L15). Should I change it? If I change it, I will have to fix all scenarios with "Given I want to create a new tax rate".

Reply:
I see now `Given` and `When` are the same thing for Behat so I've committed your suggestion. Thanks.